src/platform/packages/shared/kbn-unified-tabs/src/components/tab_preview/tab_preview.tsx

```diff
@@ -114,6 +114,12 @@ export const TabPreview: React.FC<TabPreviewProps> = ({
     };
   }, [previewTimer]);
 
+  useEffect(() => {
+    if (stopPreviewOnHover && previewTimer) {
+      clearTimeout(previewTimer);
+    }
+  }, [previewTimer, stopPreviewOnHover]);
```

Comment:
@jughosta Didn't test it, but does this fix the issue I mentioned [here](https://github.com/elastic/kibana/issues/228174#issuecomment-3079941821)? Just curious if I need to spin off a separate issue for it or not.